src/iiiftilesource.js

```diff
@@ -181,20 +182,34 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
     },
 
     /**
-     * Return the tileSize for the given level.
+     * Return the tileWidth for the given level.
      * @function
      * @param {Number} level
-    */
+     */
+    getTileWidth: function( level ) {
```

Comment:
I'm out today too; I can try to have a look over the weekend. Reviewing from my tablet, it's not immediately clear that this is accounting for the fact there can be different widths and heights at different levels, eg. http://libimages.princeton.edu/loris2/pudl0130%2F8555444%2F02%2F00000007.jp2/info.json; but maybe I'm just missing it.